static/app/components/events/groupingInfo/groupingInfo.tsx

```diff
@@ -81,12 +84,29 @@ export default function GroupingInfo({
           </div>
         )}
       </ConfigHeader>
+      <div style={{display: 'flex', justifyContent: 'flex-end', width: '100%'}}>
```

Comment:
might need a real designer's opinion on this but i feel like this should either be more aligned with the groupedby/grouping info section (not on it's own line) or left aligned. right aligned and on its own seems a little odd